src/plugins/codeStyle/index.ts

```diff
@@ -52,6 +57,13 @@ export default class CodeStyle {
             }
         }
 
+        // if (isXmlFile(file)) {
```

Comment:
Delete the commented-out code or add a leading comment with a TODO explaining why they're it's still here/what its purpose is.

Reply:
Done